session/recursive.go

```diff
@@ -0,0 +1,105 @@
+package session
+
+import (
+	"sort"
+	"strings"
+
+	"launchpad.net/gozk"
+)
+
+var defaultACL = zookeeper.ACL{
```

Comment:
I usually use `zookeeper.WorldACL(zookeeper.PERM_ALL)`. That might be a shorter way of writing this?